webServer.js

```diff
@@ -446,46 +479,49 @@ function start() {
       langPrefix: ''
     });
 
-    setCache(res, 72);
-    res.send(generatePage({
-      reqUrl: req.url,
-      title: tutorialPackage.name + ' - ' + library + ' tutorials - cdnjs.com',
-      page: {
-        template: templates.tutorial,
-        data: {
-          tute: marked(tutorialFile),
-          avatar: avatar,
-          tutorial: tutorialPackage,
-          disqus_shortname: tutorialPackage.disqus_shortname || 'cdnjstutorials',
-          disqus_url: tutorialPackage.disqus_url || ('https://cdnjs.com/' + req.originalUrl),
-          disqus_id: tutorialPackage.disqus_url || req.originalUrl,
-          author: tutorialPackage.author,
-          tutorials: tutorialPackages,
-          library: library
+    var template = templates.tutorial;
+    if (!cacheCheck(req, res, 72, template.lastModified)) {
+      res.send(generatePage({
+        reqUrl: req.url,
+        title: tutorialPackage.name + ' - ' + library + ' tutorials - cdnjs.com',
+        page: {
+          template: template.content,
+          data: {
+            tute: marked(tutorialFile),
+            avatar: avatar,
+            tutorial: tutorialPackage,
+            disqus_shortname: tutorialPackage.disqus_shortname || 'cdnjstutorials',
+            disqus_url: tutorialPackage.disqus_url || ('https://cdnjs.com/' + req.originalUrl),
+            disqus_id: tutorialPackage.disqus_url || req.originalUrl,
+            author: tutorialPackage.author,
+            tutorials: tutorialPackages,
+            library: library
+          }
         }
-      }
-    }));
+      }));
+    }
   });
 
   app.get('/libraries/:library/:version', libraryResponse);
 
   app.get('/libraries/:library', libraryResponse);
 
-  app.get('/libraries', function (req, res) {
-    setCache(res, 2);
-
-    res.send(generatePage({
-      reqUrl: req.url,
-      title: 'libraries - ' + TITLE,
-      page: {
-        template: templates.libraries,
-        data: {
-          packages: _.toArray(LIBRARIES_MAP),
-          libCount: Object.keys(LIBRARIES_MAP).length,
-          libVerCount: LIBRARIES_VERSIONS
+  app.get('/libraries', function(req, res) {
```

Comment:
Space between `function` & `(` please.

Reply:
We should probably add an `.eslintrc` file and set https://eslint.org/docs/rules/space-before-function-paren

Reply:
Good idea!

Reply:
I'll fix these up and submit a pr for that later.

Reply:
https://github.com/cdnjs/new-website/pull/280 fixes these types of things going forward.